banzai/utils/fits_utils.py

```diff
@@ -213,26 +214,28 @@ def get_basename(path):
     return basename
 
 
+@retry(wait=wait_exponential(multiplier=2, min=4, max=10), stop=stop_after_attempt(4), reraise=True)
```

Comment:
I realized these parameters aren't obvious, so I'll throw a comment in.

Stop after 4 attempts, and back off exponentially with a minimum wait time of 4 seconds, and a maximum of 10. If it fails, "reraise" the original exception back up to the caller.

Reply:
https://tenacity.readthedocs.io/en/latest/index.html#waiting-before-retrying